backend/curriculum_tracking/activity_log_entry_creators.py

```diff
@@ -64,7 +64,7 @@ def log_card_moved_to_review_feedback(card, actor_user):
 
 def log_project_competence_review_done(review):
     event_type, _ = EventType.objects.get_or_create(name=COMPETENCE_REVIEW_DONE)
-    for user in review.recruit_project.recruit_users.all():
+    for user in review.recruit_project.recruit_users.all():     # Not sure why the For loop is needed, the particular card will only have one user assigned to it
```

Comment:
I am not sure that the For loop is needed here, the recruit_project is going to be assigned to a specific user, even if it is assigned to many different users, by the time we get to "def debounce_create" part of the filtering is according to the effected_user which to my mind is the only recruit_project we are interested in seeing that the review was done on that person's recruit_project object?

Reply:
Also, from def log_project_competence_review_done(review): and from def log_topic_competence_review_done(review): we always pass debounce_seconds=False.  So then, inside def debounce_create we set debounce_seconds=120 but it seems that that doesn't matter cause when we evaluate 'if debounce_seconds' we always get False anyway so I am not sure we need this?  

What if a person does a review and then within 2 minutes that same person changes their mind and therefore the review, a new Log entry won't be created but perhaps there should be?